spec/chrono_model/adapter/base_spec.rb

```diff
@@ -134,6 +134,25 @@
       it { expect(adapter.is_chrono?(table)).to be(false) }
     end
 
+    context "when passing a fully specified schema" do
```

Comment:
Thanks for the failing spec, but to speed up things I need to better understand what it is the use case that this change is addressing.

Any chance to fork https://github.com/diowa/ruby3-rails7-bootstrap-heroku/tree/chronomodel and add a commit with the minimum amount of code needed to understand the impact of this issue on an existing application?